analytics-forms/src/main/java/com/liferay/analytics/client/android/forms/Forms.kt

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.analytics.client.android.forms
+
+import android.widget.EditText
+import com.liferay.analytics.client.android.Analytics
+
+/**
+ * @author Igor Matos
+ */
+object Forms {
+
+	private const val APPLICATION_ID = "forms"
+	private const val FIELD_NAME = "fieldName"
+	private const val FOCUS_DURATION_KEY = "focusDuration"
+	private const val FORM_ID_KEY = "formId"
+	private const val FORM_TITLE_KEY = "formTitle"
+	private const val FIELD_TITLE_KEY = "fieldTitle"
+
+	@JvmStatic
+	fun formSubmitted(formContext: FormContext) {
+		val eventId = "formSubmitted"
```

Comment:
An enum for type of events supported?

Reply:
The send works for custom events too but seems a good idea to have an alternative which receives an enum.

Reply:
I was thinking more in that if you are inside a form.kt class and want to have explicitely the events supported, instead of finding the methods or the strings, having an enum that list them all (and can be translated to the string)